fcrepo-kernel-impl/pom.xml

```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.fcrepo</groupId>
+    <artifactId>fcrepo</artifactId>
+    <version>6.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>fcrepo-kernel-impl</artifactId>
+  <name>Fedora Repository Kernel Implementation</name>
+  <description>Fedora Repository Kernel Implementation, containing implementation of core kernel API interfacess</description>
+
+  <packaging>bundle</packaging>
+
+  <properties>
+    <osgi.import.packages>
+      *
+    </osgi.import.packages>
+    <osgi.export.packages>
+      org.fcrepo.kernel.*;version=${project.version}
```

Comment:
I'm not strong with OSGI, but if `fcrepo-kernel-api` exports `org.fcrepo.kernel.api` should this export `org.fcrepo.kernel.impl`?